src/actions/__tests__/habits.spec.js

```diff
@@ -1,14 +1,26 @@
 import * as actions from './../habits';
-import { status } from './../../constants/habit';
+import { status, Habit } from './../../constants/habit';
 
 describe('habit actions', () => {
   it('should create an habit with correct type', () => {
     expect(actions.addHabit('Fumar oregano').type).toBe('add_habit');
   });
+
   it('should create an habit with status ACTIVE', () => {
     expect(actions.addHabit('Fumar oregano').payload.status).toBe(status.ACTIVE);
   });
+
   it('should create an habit with a name', () => {
     expect(actions.addHabit('Fumar oregano').payload.name).toBe('Fumar oregano');
   });
+
+  it('should get type "see_habit" when invoque "seeHabit" action', () => {
```

Comment:
should receive seen_habit action

Reply:
La prueba aquí es lo que retorna la función 'seenHabit', no lo que recibe.  Como lo que retorna es un objeto que contiene un 'type' y un 'payload'. Se prueba que se obtiene type = 'seen_habit'